protocol/messenger_contacts.go

```diff
@@ -170,11 +170,14 @@ func (m *Messenger) dismissContactRequest(requestID string, syncing bool) (*Mess
 	}
 
 	if notification != nil {
-		err := m.persistence.UpdateActivityCenterNotificationMessage(notification.ID, contactRequest)
-		if err != nil {
-			return nil, err
-		}
 		notification.Message = contactRequest
+		notification.Read = true
+
+		saveErr := m.persistence.SaveActivityCenterNotification(notification)
+		if saveErr != nil {
+			m.logger.Warn("failed to save notification", zap.Error(saveErr))
```

Comment:
I'm wondering if we should log as `Error` instead of `Warn`. It seems to me the errors that can happen inside `SaveActivityCenterNotification` should rarely happen, like marshaling a message, but when they do it's because something weird is going on, so better report as error. Wdyt?

Reply:
fixed